source/_assets/stylesheets/base/_lists.scss

```diff
@@ -3,6 +3,7 @@ ol {
   margin: 0;
   padding: 0;
   list-style-type: none;
+  overflow: auto; //clearfix
```

Comment:
on mobile, floated lis on prizes and judges are all bunched up. this fixes it but should I have not touched /base/*?